### PR TITLE
Update roadmap view

### PR DESCRIPTION
**Pull request by maleeha13**
Implementing design changes: https://www.figma.com/file/TKqqDs89PXvn3L5xH2f96V/Roadmap-Page?type=design&node-id=9-18925&mode=design&t=wL8UQJDblhkbchGT-0

### CODE REVIEW

File: site/src/components/roadmap/style.module.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  margin-bottom: 15px;
+  min-height: unset;
+  border-left-color: var(--salt-palette-neutral-secondary-foreground);
+}
+
+.cardInProgress {
+  width: 400px;
+  margin-bottom: 15px;
+  min-height: unset;
+  border-left-color: var(--salt-palette-info-border);
```
Comment (libertymayc) [inline site/src/components/roadmap/style.module.css:51]:
```suggestion
  border-left-color: var(--salt-status-info-borderColor);
```
characteristics should be used directly, never palette

File: site/src/components/roadmap/style.module.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.backlogIcon {
+  color: var(--salt-palette-neutral-secondary-foreground);
+  margin-right: 10%;
+}
+
+.progressIcon {
+  color: var(--salt-palette-info-border);
```
Comment (libertymayc) [inline site/src/components/roadmap/style.module.css:60]:
```suggestion
  color: var(--salt-status-info-borderColor);
```

File: site/src/components/roadmap/style.module.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+.cardInProgress {
+  width: 400px;
+  margin-bottom: 15px;
+  min-height: unset;
+  border-left-color: var(--salt-palette-info-border);
+}
+
+.backlogIcon {
+  color: var(--salt-palette-neutral-secondary-foreground);
```
Comment (libertymayc) [inline site/src/components/roadmap/style.module.css:55]:
```suggestion
  color: var(--salt-status-static-foreground);
```

File: site/src/components/roadmap/style.module.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+.column {
+  padding-right: 20px;
+}
+
+.cardFuture {
+  width: 400px;
+  margin-bottom: 15px;
+  min-height: unset;
+  border-left-color: var(--salt-palette-neutral-secondary-foreground);
```
Comment (libertymayc) [inline site/src/components/roadmap/style.module.css:44]: this token doesnt fit here, needs looking at. potentially separable secondary border?

Comment (maleeha13) [reply site/src/components/roadmap/style.module.css]: changed it to --salt-status-static-foreground to match the icon.

**Review by navkaur76 — COMMENTED** [review]
I would double check some of the horizontal spacing values between content, e.g. the spacing directly above the filter input is looking a little tight compared to the figma file (which is 36px).

**Review by joshwooding — APPROVED** [review]
I think this is good enough for now

**Review by libertymayc — APPROVED** [review]

**Pull request merged**